src/plotting.jl

```diff
@@ -0,0 +1,413 @@
+    #TODO: mean predicted value histogram underneath?? Maybe important...
+    # https://scikit-learn.org/stable/modules/calibration.html
+    linesegments!(ax, [0, 1], [0, 1]; color=(:black, 0.5), linewidth=2, linestyle=:dash, label="Ideal")
+    return ax
+end
+
+function plot_binary_discrimination_calibration_curves!(subfig::FigurePosition, calibration_curve::SeriesCurves, calibration_score,
+                                                        per_expert_calibration_curves::SeriesCurves,
+                                                        per_expert_calibration_scores, optimal_threshold,
+                                                        discrimination_class::AbstractString;
+                                                        markershape=Rect, markersize=5)
+    ax = plot_pr_curves!(subfig, per_expert_calibration_curves, nothing; legend=nothing,
+                         title="Detection calibration", xlabel="Expert agreement rate",
+                         ylabel="Predicted positive probability", color=:darkgrey,
+                         scatter=(markershape=markershape, markersize=markersize))
+
+    scatter = (markershape=:circle, markersize=markersize, markerstrokewidth=0, color=:navyblue)
+    series!(ax, calibration_curve; scatter=scatter, color=:navyblue, linewidth=1)
+    linesegments!(ax, [0, 1], [0, 1]; color=(:black, 0.5), linewidth=2, linestyle=:dash, label="Ideal")
+    #TODO: expert agreement histogram underneath?? Maybe important...
+    # https://scikit-learn.org/stable/modules/calibration.html
+    return ax
+end
+
+function plot_confusion_matrix!(subfig::FigurePosition, confusion::NumberMatrix,
+                                class_labels::AbstractVector{String},
+                                normalize_by::Symbol;
+                                annotation_text_size=20,
+                                colormap=:Blues)
+    normdim = get((Row=2, Column=1), normalize_by) do
+        return error("normalize_by must be either :Row or :Column, found: $(normalize_by)")
+    end
+
+    nclasses = length(class_labels)
+    if size(confusion) != (nclasses, nclasses)
+        error("Labels must match size of square confusion matrix. Found $(nclasses) labels for an $(size(confusion)) matrix")
+    end
+    confusion = round.(confusion ./ sum(confusion; dims=normdim); digits=3)
+    class_indices = 1:nclasses
+    max_conf = maximum(confusion)
+    ax = Axis(subfig;
+              title="$(string(normalize_by))-Normalized Confusion",
+              xlabel="Elected Class",
+              ylabel="Predicted Class",
+              xticks=(class_indices, class_labels),
+              yticks=(class_indices, class_labels),
+              xticklabelrotation=pi / 4)
+
+    hidedecorations!(ax, label = false, ticklabels = false)
+
+    ylims!(ax, nclasses, 0)
+
+    tightlimits!(ax)
+    # Really unfortunate, that heatmap is not correctly aligned
```

Comment:
:|

Reply:
On the upside, this is fixed in the newest release ;)